src/server.js

```diff
@@ -15,7 +17,23 @@ const ERROR_BAD_DB_INTERACTION = "BAD_DATABASE";
 const ERROR_INVALID_EVENT_ID = "INVALID_EVENT_ID";
 const ERROR_BAD_UUID = "BAD_UUID";
 
-app.use(express.static("static"));
+app.use(express.static("static/absolute"));
```

Comment:
The reason we are serving the HTML files individually is because if we use `express.static` to serve the HTML files, then we will only see the participants page at "ogene.com/participants.html". This is not good because we want to serve the participants page at "ogene.com/<event id>/participants.html".

That's why the HTML files have specific routes, like `/:${URL_PARAM_EVENT_ID}/participants`.

However, we don't want to serve the css/js at `ogene.com/<event id>/participants.js` (well we could, but it seems unnecessary), so we serve it absolutely at `ogene.com/js/participants.js`